lib/Reader.php

```diff
@@ -39,6 +39,10 @@ public function getClark()
             return null;
         }
 
+        if (is_null($this->namespaceURI) || empty($this->namespaceURI)) {
```

Comment:
I guess this can be simplified to
```suggestion
        if (empty($this->namespaceURI)) {
```